webapp/portlet/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue

```diff
@@ -5,12 +5,12 @@
     py-0
     class="white d-none d-sm-block">
     <v-row v-if="navigationTree && navigationTree.length" class="mx-0 administrationTitle">
-      <v-list-item @mouseover="openDrawer()" @click="openDrawer()">
+      <v-list-item>
         <v-list-item-icon class="mb-2 mt-3 mr-6 titleIcon"><i class="uiIcon uiIconToolbarNavItem uiAdministrationIcon"></i></v-list-item-icon>
         <v-list-item-content class="subtitle-2 titleLabel clickable">
           {{ this.$t('menu.administration.title') }}
         </v-list-item-content>
-        <v-list-item-action class="my-0">
+        <v-list-item-action class="my-0" @click="toggleOpenDrawer()">
```

Comment:
To place on `<v-list-item>` to make the action triggered on all element @SaraBoutej @AzmiTouil . ( feedback emitted to the Team )